notebooks/image_models/solutions/kfp_image_classification.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.    def create_pipeline():
</pre><p> </p><p>What do you think about adding a data preprocessing component that calls Dataflow job to create TFRecords?</p><p>(It is equivalent to this notebook you created: https://github.com/GoogleCloudPlatform/asl-ml-immersion/pull/224)</p><p> </p><p>This notebook will be the used in the last module of Day 4, and TFRecords topic is already covered in Day 2, so it would be better to have it in the pipeline.</p><p> </p><p>Reference:</p><p>It is similar to a component of this notebook from the cv book, but I think it is a bit buggy...</p><p>https://github.com/GoogleCloudPlatform/practical-ml-vision-book/blob/master/10_mlops/10a_mlpipeline.ipynb</p><p> </p><p>This is the fixed version of it (Sorry for Japanese descriptions)</p><p>https://github.com/takumiohym/practical-ml-vision-book-ja/blob/main/10_mlops/10a_mlpipeline.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>